katdal/test/test_chunkstore.py

```diff
@@ -241,3 +249,14 @@ def test_dask_array_put_parts_get_whole(self):
         # Now store the last quarter and check that complete array is correct
         self.put_dask_array('big_y2', np.s_[3:8, 30:60, 0:2])
         self.get_dask_array('big_y2')
+
+    def test_list_of_chunk_ids(self):
```

Comment:
Remove `of` so that test name matches the method being tested.

Reply:
Good idea.